pkgs/top-level/all-packages.nix

```diff
@@ -13290,6 +13290,11 @@ with pkgs;
 
   adobe-reader = callPackage_i686 ../applications/misc/adobe-reader { };
 
+  masterpdfeditor = callPackage ../applications/misc/masterpdfeditor {
```

Comment:
Ok that fills in the qtbase and qtsvg dependencies, but something is still broken with xcb. It didn't happen with nix-shell.